src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CartRuleController.php

```diff
@@ -24,13 +24,27 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+declare(strict_types=1);
+
```

Comment:
I still see "declare(strict_types=1);" here or when I click to view file.
But I don't see when I try to edit the file trough github, or edit file trough my IDE so this confuses me.

Reply:
The file you're watching seems to be outdated :thinking:  I don't see it when I check "View File"